Assets/MixedRealityToolkit.Examples/Experimental/Demos/UX/BoundsControl/Scripts/BoundsControlRuntimeExample.cs

```diff
@@ -0,0 +1,253 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using Microsoft.MixedReality.Toolkit.Input;
+using Microsoft.MixedReality.Toolkit.UI;
+using Microsoft.MixedReality.Toolkit.Experimental.UI.BoundsControl;
+using Microsoft.MixedReality.Toolkit.Utilities;
+using System.Collections;
+using System.Text;
+using TMPro;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.Examples.Experimental.Demos
+{
+    /// <summary>
+    /// TODO: This demo probably needs to be adjusted
```

Comment:
[Is this TODO tracked elsewhere as well](https://microsoft.github.io/MixedRealityToolkit-Unity/Documentation/Contributing/DocumentationGuide.html#todos)?

Reply:
https://github.com/microsoft/MixedRealityToolkit-Unity/issues/6926 added ticket for cleaning up this class and added to bounding box refactor task - i don't want to clean it up for now because I might rewrite it several times until we reach the final version. Just copied this over so we don't miss any of the tests we already had in place for the old bounding box